index.html

```diff
@@ -9,39 +9,290 @@
+            <div>
+                <h1 class="cardonept">
+                    Multi-Post
+                    Stories
+                </h1>
+                <ul class="frame2">
+                    <li id="listid">CANOPY</li>
+                    <!-- <img src="img/Counter.png" alt=""> -->
+                    <li class="lineitem">Back End Dev</li>
+                    <!-- <img src="img/Counter.png" alt=""> -->
+                    <li class="lineitem">2015</li>
+                </ul>
+
+                <p class="tagtext">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+
+                <ul class="frame2">
+                    <li class="tag">html </li>
+                    <li class="tag">css</li>
+                    <li class="tag">javaScript</li>
+                </ul>
+
+                <div>
+                    <div class="action">
+                        <button class="buttonaction">See Project</button>
+                    </div>
+                </div>
+
+            </div>
+
+
+
+        </section>
+
+        <section class="cardone">
+            <img src="img/Snapshoot Portfolio-2.png" alt="">
+            <div>
+                <h1 class="cardonept">
+                    Tonic
+                </h1>
+                <ul class="frame2">
+                    <li id="listid">CANOPY</li>
+                    <!-- <img src="img/Counter.png" alt=""> -->
+                    <li class="lineitem">Back End Dev</li>
+                    <!-- <img src="img/Counter.png" alt=""> -->
+                    <li class="lineitem">2015</li>
+                </ul>
+
+                <p class="tagtext">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. </p>
+
+                <ul class="frame2">
+                    <li class="tag">html </li>
+                    <li class="tag">css</li>
+                    <li class="tag">javaScript</li>
+                </ul>
+
+                <div>
+                    <div class="action">
+                        <button class="buttonaction">See Project</button>
+                    </div>
+                </div>
+
+            </div>
+
+
+
+        </section>
+    </section>
+
+
+
+
+
+    <section class="aboutandcontact">
+        <section class="uniquecard">
+            <main id="unique">
+
+                <h1 class="aboutheader">
+                    About
+                    <br>
+                    Myself
+                </h1>
+                <p class="aboutbody">Hello I’m a software developer! I can help you build a product , feature or
+                    website
+                    Look through some of my work and experience! If you like what you see and have a project you need
+                    coded,
+                    don’t hestiate to contact me.
+                </p>
+
+                <a href="microverse.org" class="connect">LET’S CONNECT</a>
```

Comment:
- [ ] Please make sure you use flexbox to align the content of your about myself section, for the layout of this section you should use flexbox as we need you to be a flex ninja :ninja: It is also a very important skill for frontend develoment,
- [ ] currently you have used flex only on `ul` and `li` items.